default/tests/test_main.py

```diff
@@ -10,3 +10,22 @@ def test_index():
     assert response.status_code == 200
     with open('tests/testfiles/index.html', 'r') as correct:
         assert response.text == correct.read()
+
+
+def test_userdata():
```

Comment:
Add more testing:
* What happens if there are missing parameters
* What happens if any of the parameters is incorrect
* etc